contents/blog/why-redesign.mdx

```diff
@@ -0,0 +1,171 @@
+
+### 3. Flexibility
+
+Some of our customers use all our products. Some only use one or two. The experience should feel like it was designed specifically for however you use PostHog.
+
+## The biggest changes you'll see today
+
+This initial release of our updated UI is largely focused on aesthetic changes, though it does come with some new features.
+
+But more importantly, it lays the groundwork for future improvements. **Some things may still feel a bit janky, but there are lots of exciting plans. This is a work in progress, and I hope you'll bear with us as we continue to iterate in the coming months.**
+
+Here's some of the design changes (and a few new features) you'll see today:
+
+### 1. Colors, data density, dark mode
+
+_Data density, dark mode, developers._ Now you can see more information on the screen, and it won't make your eyes bleed.
+
+You can change between light, dark, and a mode that will match your system preference under the account menu.
+
+![PostHog 3000 in dark mode](../images/blog/why-redesign/dark-mode.png)
+
+### 2. New navigation menu grouping
+
+<div class="sm:max-w-sm sm:float-right -mt-12 -mb-12 sm:-ml-8">
+
+![PostHog 3000 nav menu](../images/blog/why-redesign/nav.png)
+
+</div>
+
+The nav menu has now been organized into two main sections: **project & data** and **products**.
+
+**Project & data** includes your dashboards and notebooks, along with the data you're sent to PostHog (like events, people, and properties), and metadata you've added on top (like actions, cohorts, and annotations) – and a live feed of incoming events.
+
+**Products** is the ever-growing list of tools built on top of the PostHog platform.
+
+Once the icons are familiar enough, you may want to hide the text labels. You can do this by dragging the dividing line to the left of the screen (or just double-click it).
+
+In the future, we may add the [ability to customize the nav menu](https://github.com/PostHog/posthog/issues/18884) by re-ordering items or hiding products you don't use.
+
+<div class="clear-both">&nbsp;</div>
+
+### 3. Side panel
+
+The new collapsible side panel is always available on the right side of the screen and is designed to help with whatever you're currently doing.
+
+- **Notebooks:** Drag almost anything from the main workspace into a notebook so you don't lose it. It's designed to be a scratchpad for data exploration.
+- **In-app docs (seen below):** Clicking any docs link in the product will now open in the side panel, keeping you in the flow of what you're doing.
+- **Support:** Get in touch with support without having a model blocking whatever it is you were looking at.
+
+![PostHog 3000 in dark mode](../images/blog/why-redesign/side-panel.gif)
+
+The side panel may also open other panes as necessary – like quick access to settings, or an onboarding checklist if you're new to PostHog.
+
+### 4. Notebooks
```

Comment:
I feel strongly that notebooks should be higher up this list. The most impactful changes for users are: Dark Mode, Notebooks, Side panel, in that order.